src/api/dir.h

```diff
@@ -28,12 +28,12 @@ enum nspire_dir_type {
 
 struct nspire_dir_item {
 	char name[240];
-	unsigned long size, date;
+	uint64_t size, date;
```

Comment:
`size_t` for size, not sure what `date` should be. Internally both are `uint32_t`.